pkg/broker/recover_broker_info.go

```diff
@@ -0,0 +1,56 @@
+package broker
+
+import (
+	"context"
+	"encoding/base64"
+	"github.com/submariner-io/admiral/pkg/reporter"
+	"github.com/submariner-io/subctl/internal/constants"
+	"github.com/submariner-io/subctl/internal/rbac"
+	"github.com/submariner-io/subctl/pkg/cluster"
+	"github.com/submariner-io/submariner-operator/api/v1alpha1"
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func RecoverData(
```

Comment:
There are reasons for this:

1. `broker.WriteInfoToFile` everytime generates a new PSK secret which we don't want. It would need to be tweaked and **can't be used as is**. 
2. It expects components, customDomains as arguments. So to get these from Broker, a new function would anyways need to be written to fetch them first and then pass to `broker.WriteInfoToFile`.
3. It expects `ipsec File` (file that contains PSK Secret specified by the user) as an argument. We are not providing this support via `recover-broker-info` command
4. `recover-broker-info` is a new command and it makes sense that it goes to its dedicated function call

I have reused `data.writeToFile` that ``broker.WriteInfoToFile` also uses to write the collected data to `broker-info.subm`

Reply:
See https://github.com/submariner-io/subctl/pull/588.

Reply:
Should we fold it into this one, or treat as standalone PR?

Personally, I think squashing would be better.

Reply:
I’d rather keep them separate, let’s get this one merged (once the remaining comments are addressed) and finish reviewing #588 separately.